flow/activities/flowable.go

```diff
@@ -178,6 +180,14 @@ func (a *FlowableActivity) handleSlotInfo(
 		return err
 	}
 
+	slotLagInMBThreshold := peerdbenv.GetPeerDBSlotLagMBAlertThreshold()
+	if uint32(slotInfo[0].LagInMb) >= slotLagInMBThreshold {
```

Comment:
add a check for if the number of open connections for the peerdb user on postrges is also greater that configurable threshold, make it 5 to start with.